### ISSUE TITLE
Legend for raw scores is hard to parse

### ISSUE THREAD

**Issue by riecke** [OurPlanscape/Planscape#417]
From Nick P: "I think the legend should reflect the actual scale of the data when the raw data values are selected as opposed to the "normalized" values. I can't tell if the "Current Condition" is a score or if it's the raw data. That should be expressly stated though and don't think you can get it done with a shared legend."

Some ideas:
1. Change the legend so that it's not "Poor --> Excellent" but "Low --> High" or perhaps the values themselves.
2. Change the legend to reflect the map under focus, using a different title for raw condition scores.

**Comment by mskim3494** [comment]
Decisions from product meeting:
- For raw data layers from RRKs, [min, max] to be put on the legend (on each corresponding color). This achieves two things: distinguish the type of data that we're looking at, and makes it easier for the user to mentally map the color to the value (less UI interaction)
- Eng to investigate whether color *gradient* for heatmaps can be exposed, as opposed to color buckets, for both types of data (raw and normalized)
